vignettes/intro_tutorial.qmd

```diff
@@ -338,23 +321,28 @@ with(
 )
 ```
 
+**tinyplot** also supports special types to fit models and display their predictions, along with confidence intervals. Here's an example where we fit a logistic regression model:
+
+```{r ribbon_pred}
+with(aq,
+  tinyplot(
+    I(Temp > 80) ~ Wind,
+    type = type_glm(family = binomial))
+)
+```
```

Comment:
Could you perhaps add a sentence or two along the lines of:

"We'll see other examples of additional plot types below, including a few more of the special model prediction types.  But please also take a look at the dedicated [Plot types](https://grantmcdermott.com/tinyplot/vignettes/types.html) vignette for explicit details on all of the different plot types that **tinyplot** supports, as well as how to roll your own custom types."